mods/ts/sequences/infantry.yaml

```diff
@@ -848,7 +846,6 @@ chamspy:
 	die6: electro
 		Length: *
 		Tick: 80
-	icon: chamicon
```

Comment:
Wasn't buildable in the original and the icon is already used by the buildable hijacker (in the original, too), so this removal is correct.